util/io/bytereadseeker.go

```diff
@@ -33,6 +32,5 @@ func (f byteReadSeeker) Seek(offset int64, whence int) (int64, error) {
 	if offset < 0 || offset > int64(len(f.data)) {
 		return 0, &fs.PathError{Op: "seek", Err: fs.ErrInvalid}
 	}
-	f.offset = offset
```

Comment:
Ops, this looks like a real bug , is not it? I think we should change `byteReadSeeker` to `*byteReadSeeker` instead.

Reply:
Definitely could be a real bug... will look at what calls these methods.

Reply:
I wonder if we've just been lucky up to this point. Maybe index.html is small enough that it all fits into one read.

Reply:
Yeah, I think Seek just never been called so far. But it might be called in future